src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -295,6 +295,10 @@ class CollapsibleCourseRun extends React.Component {
           disabled={courseInReview}
           required={courseRunSubmitting}
         />
+        <div>
+          <FieldLabel id="number" text="Add OFAC Restriction text to the FAQ section" className="mb-2" />
+          <div className="mb-3">{courseRun.has_ofac_restrictions ? 'Yes' : 'No'}</div>
```

Comment:
This is meant to be just a plain text display and not part of the form right? Just verifying.

Reply:
Yes, not meant to be editable by partners and we have not added the form actions yet.